mongodb_migrations/cli.py

```diff
@@ -22,23 +22,24 @@ def __init__(self):
 
     def create_config(self):
         self.config = Configuration()
-        self.config.from_console()
         if self.config.migrator_config != None:
             self.config.from_ini()
+        else:
+            self.config.from_console()
+
 
     def run(self):
         self.db = self._get_mongo_database(self.config.mongo_host, self.config.mongo_port, self.config.mongo_database)
         files = os.listdir(self.config.mongo_migrations_path)
-        migrations = []
+        migrations = {}
         for f in files:
-            result = re.match('^\d+\.py$', f)
+            result = re.match('^(\d+)[_a-z]*\.py$', f)
```

Comment:
Having a short description is a great idea!

Reply:
Thanks :) got this thing from one of the ruby frameworks